.devcontainer/Dockerfile

```diff
@@ -13,6 +13,12 @@ RUN poetry config virtualenvs.create false
 
 WORKDIR /workspaces/runner-manager
 
+# Create a non-root user
+
```

Comment:
In this image, it's not used in runtime, I would say we can remove any change and solve the security issue with "used in tests"

Reply:
Oh ok, I see, you're right, we can do this

Reply:
Done : https://github.com/scality/runner-manager/security/code-scanning/72